README.md

```diff
@@ -211,6 +212,8 @@ will be used if present, else we fall back on the application's launcher icon.
         <!-- ... -->
         <meta-data android:name="com.klaviyo.push.default_notification_icon"
             android:resource="{YOUR_ICON_RESOURCE}" />
+        <meta-data android:name="com.klaviyo.push.default_notification_color"
```

Comment:
Where'd the idea to add colour come from?

Reply:
its been in the backlog, it was raised as a github issue #144.

Reply:
Ah I see, issue https://github.com/klaviyo/klaviyo-android-sdk/issues/104
Cool!

Reply:
whoops yeah wrong number